packages/create-llama/questions.ts

```diff
@@ -420,16 +420,18 @@ export const askQuestions = async (
 
   if (program.framework === "nextjs") {
     const { observability } = await prompts({
-      type: "toggle",
+      type: "select",
       name: "observability",
       message: "Would you like to set up observability?",
-      initial: getPrefOrDefault("observability"),
-      active: "Yes",
-      inactive: "No",
+      choices: [
+        { title: "No", value: "none" },
+        { title: "OpenTelemetry", value: "OpenTelemetry" },
```

Comment:
as far as I understood it, there might be other options?
```suggestion
        { title: "OpenTelemetry (Using Traceloop)", value: "OpenTelemetry" },
```

Reply:
@marcusschiesser no, since we output standard OpenTelemetry so this can be connected to any platform (https://www.traceloop.com/docs/openllmetry/integrations/introduction)